topics/cd/index.md

```diff
@@ -0,0 +1,9 @@
+---
+display_name: CD (Disambiguation)
+short_description: YAML is a human-friendly data serialization standard.
```

Comment:
This looks like a copy-paste leftover?

Reply:
Damn, it is 😅 Glad, one of us noticed it 😬